examples/platform/silabs/main.cpp

```diff
@@ -48,6 +49,19 @@ int main(void)
 {
     GetPlatform().Init();
 
+    main_Task = osThreadNew((osThreadFunc_t)application_start, NULL, &thread_attributes);
```

Comment:
I am ok with this, if we have t keep what you added in `SilabsMatterConfig::InitMatter` (matterConfig.cpp) But I believe we can work something in the platform abstraction init for that.

Reply:
I am not finding any other solution .

I have changed the code in SilabsMatterConfig::InitMatter   slightly , please have a look

Reply:
That should be put inside the SPAM Init for the WiseConnect like Junior stated under /src/platform/silabs/platformabstraction

Reply:
SPAM init is called before we start the scheduler.

Junior is aware about our limitation that we need to call " sl_wifi_init" only from thread context.

That's why we cannot call it inside SPAM init in platformabstraction